### PR TITLE
Update config.yml

### PR DESCRIPTION
**Pull request by aroramrinaal**
# Added the missing endline.

## Description:
Previous commit at #5 did not had the endline.

## Related Issue:
#3 

Closes #XXXXX

### CODE REVIEW

**Review by aroramrinaal — COMMENTED** [review]
Fixed the missing endline on #5

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -2,4 +2,5 @@ blank_issues_enabled: false
 contact_links:
   - name: Questions
     url: https://discord.gg/Byjr6rdBUZ
-    about: Any questions or comments you have, let us know on our Discord server.
\ No newline at end of file
+    about: Any questions or comments you have, let us know on our Discord server.
+
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/config.yml]: Looks like there are _two_ extra lines now. GitHub doesn't display the endline.
```suggestion
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (aroramrinaal) [reply .github/ISSUE_TEMPLATE/config.yml]: I just added a single extra line don't know why it shows _two_ extra lines.

Comment (Panquesito7) [reply .github/ISSUE_TEMPLATE/config.yml]: Hmm, well, GitHub shows an extra line there. It looks like it has a trailing whitespace.

Comment (aroramrinaal) [reply .github/ISSUE_TEMPLATE/config.yml]: Screenshot 2022-06-18 at 9 57 44 AM

**Pull request closed without merging**